src/style.css

```diff
@@ -0,0 +1,113 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap");
+
+body {
+  font-family: "Roboto", sans-serif;
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  width: 166px;
+  border: 1px solid #f3f3f3;
+  background: #fff;
+  border-radius: 5px;
+  padding: 32px 16px 16px ;
+}
+
+.card__image {
+  width: 160px;
+  height: 134px;
+  margin: auto;
+}
+
+.name {
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  color: #060b35;
+  margin: 40px 0 0;
+}
+
+.code {
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: #616070;
+  margin: 4px 0 0;
+}
+
+.stars {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  height: 16px;
+  margin-top: 16px;
+}
+
+.stars_star {
+  background-image: url(images/star.svg);
+  background-repeat: no-repeat;
+  background-size: cover;
+  width: 12px;
+  height: 12px;
+  margin: 2px 6px 2px 2px;
+}
+
+.stars :nth-child(-n+4) {
+  background-image: url(images/star-active.svg);
+}
+
+.review {
+  text-align: center;
+  font-weight: 400;
+  font-size: 10px;
+  margin: auto 0 0 auto;
+}
+
+.price {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  height: 18px;
```

Comment:
why give it a height? what if i have more text?

Reply:
fixed